src/NHibernate/Connection/ConnectionProvider.cs

```diff
@@ -203,5 +204,11 @@ protected virtual void Dispose(bool isDisposing)
 		}
 
 		#endregion
+
+		//TODO 6.0: Make abstract
+		public virtual DbConnection GetConnection(string connectionString)
```

Comment:
It should be move up to where `public abstract DbConnection GetConnection();` is declared, and I think we should now provide a default implementation for this `public abstract DbConnection GetConnection();`:
```c#
		public virtual DbConnection GetConnection()
		{
			return GetConnection(ConnectionString);
		}
```

Reply:
Done